modules/brushingandlinking/src/brushingandlinkingmanager.cpp

```diff
@@ -413,7 +413,7 @@ void BrushingAndLinkingManager::serialize(Serializer& s) const {
             s.serialize(toString(action), map, "selection", {},
                         [](BrushingTarget t) { return t.getString(); }, {});
         } else if (std::holds_alternative<IndexListTargets>(targetmap)) {
-            auto& map = std::get<BitSetTargets>(targetmap);
+            auto& map = std::get<IndexListTargets>(targetmap);
```

Comment:
Using get_if (https://en.cppreference.com/w/cpp/utility/variant/get_if) here would avoid writing the type twice or maybe using a visitor...